package.json

```diff
@@ -3,15 +3,23 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "aws-sdk": "^2.578.0",
```

Comment:
what are aws-sdk, child_process, cookie-parser, fs, tls? if not being used, npm uninstall them - don't want clutter